File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        
        // sorting
        sort(nums.begin(), nums.end());
        // for loop for i, i<nums.size()-1
        for(int i = 0; i<nums.size()-1;i++) {
            // if statement, nums[i + 1]
            if(nums[i]== nums[i + 1]) {
                // return true
                return true;
            }
            
        }
            //return false for else statement
            return false;
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # Time - O(n)  space - O(n)
         hashset = set()

         for n in nums:
            if n in hashset:
                return True
            hashset.add(n)
         return False
        